src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -54,7 +54,7 @@ public function load(array $configs, ContainerBuilder $container)
         $configuration = new Configuration();
         $config = $this->processConfiguration($configuration, $configs);
 
-        $container->setAlias('api_platform.naming.resource_path_naming_strategy', $config['naming']['resource_path_naming_strategy']);
+        $container->setAlias('api_platform.operation_path_resolver.default', $config['default_operation_path_resolver']);
```

Comment:
Yes, the problem is that i don't think we should replace the user service but if you think we should then it's fine for me ^^